tools/HowToUseFormatter.md

```diff
@@ -0,0 +1,43 @@
+# How to use formatter
+Here are two formatter (or beautifuler) for C and C++ languages, cformatter.sh and cxxformatter.sh.
+
+## Prerequisites
+```bash
+sudo apt-get install indent astyle
+sudo apt-get install clang-format
+```
+
+## formatter for C
+The **cformatter** script applies TizenRT C coding rule.  
+```--help``` shows how to use it.
+```bash
+@ubuntu:~/TizenRT$ ./tools/cformatter.sh --help
+usage: ./tools/cformatter.sh TARGET
+     TARGET: folder path or target files
+Apply TizenRT C coding rule
+
+example:
+    ./tools/cformatter.sh ../kernel
+    ./tools/cformatter.sh ../../abc.c
+```
+Note! This sometimes makes incorrect changes.  
+Ususally it is a space between type casting and variable as show below:
```

Comment:
Ususally ---> Usually
 "Usually it adds a space between type casting and variable as show below"